services/galley/src/Galley/Data.hs

```diff
@@ -382,7 +384,8 @@ createConversation usr name acc role others tinfo mtimer = do
             addPrepQuery Cql.insertConv (conv, RegularConv, usr, Set (toList acc), role, fromRange <$> name, Just (cnvTeamId ti), mtimer)
             addPrepQuery Cql.insertTeamConv (cnvTeamId ti, conv, cnvManaged ti)
     mems <- snd <$> addMembersUnchecked now conv usr (list1 usr $ fromConvSize others)
-    return $ newConv conv RegularConv usr (toList mems) acc role name (cnvTeamId <$> tinfo) mtimer
+    -- TODO: Should there be a default for the ReceiptMode ?
```

Comment:
If there was, lots of `Maybe`s should go away.

But I actually think we can remove this `TODO`: `Nothing` is the default, and it's the client's responsibility to interpret that.

Reply:
👌